pkg/cataloghtml/css.css

```diff
@@ -102,6 +102,15 @@ body > input {
 	min-height: 100%;
 }
 
+code {
+	overflow-x: auto;
+	white-space: pre-wrap;
+	white-space: -moz-pre-wrap;
+	white-space: -pre-wrap;
+	white-space: -o-pre-wrap;
+	word-wrap: break-word;
```

Comment:
Is it necessary to include browser-support for Mozilla back to 1999, Opera 4-7 and IE 5.5+? 😅 Or can we live with just `white-space: pre-wrap` for css-3, which is supported by all modern browsers? 

I guess I kinda like `white-space: pre-line` a bit more, but depends of course on what it will be used for. [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/white-space) has a really nice playground for testing the differences.

Reply:
Ah shoot, this comment was meant for row 107-110

Reply:
Yup this was just some copypaste fail, thanks for the input. I just pre-wrap/pre-line should be good 👍